sdk/communication/communication-chat/src/generated/src/operations/chat.ts

```diff
@@ -239,11 +239,6 @@ const listChatThreadsNextOperationSpec: coreClient.OperationSpec = {
       isError: true
     }
   },
-  queryParameters: [
-    Parameters.maxPageSize,
-    Parameters.apiVersion,
-    Parameters.startTime
```

Comment:
Please check why are these removed.

Reply:
These have been removed in the beta release. See the discussion thread: https://teams.microsoft.com/l/message/19:344f6b5b36ba414daa15473942c7477b@thread.skype/1677716157670?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1677716157670&teamName=Azure%20SDK&channelName=Language%20%E2%80%93%20JS%E2%80%89%EF%BC%86%E2%80%89TS%20%F0%9F%A5%B7&createdTime=1677716157670